src/Model/EventModel.php

```diff
@@ -40,10 +41,11 @@ public function __construct(array $config ) {
         );
         $this->properties = $config['properties'];
         // Can pass in unix timestamp if prefixed with '@'. Else just let it parse the date
-        $this->time = !empty($config['time']) ?
-            new DateTime(
-                is_int($config['time']) ? '@' . $config['time'] : $config['time']
-            ) : null;
+        $time = (array) new DateTime(
+            is_int($config['time']) ? '@' . $config['time'] : $config['time']
+        );
+
+        $this->time = !empty($config['time']) ? $time['date'] : null;
```

Comment:
I could be wrong but pretty sure we need to convert this to a unix timestamp in order to successfully send to the track api. we can either do this right here or can we make that conversion in the jsonSerialize method